spec/forms/support_interface/country_form_spec.rb

```diff
@@ -16,15 +14,27 @@
       )
     end
 
-    it { is_expected.to validate_presence_of(:eligibility_enabled) }
+    let(:country) { create(:country) }
+
+    it do
+      expect(subject).to validate_presence_of(
+        :eligibility_enabled,
+      ).with_message(
```

Comment:
I've followed the documentation when adding in with_message

https://github.com/thoughtbot/shoulda-matchers/blob/6f4de5ad6d0d09486c2c713fb1a0bf38169d399e/lib/shoulda/matchers/active_model/validate_presence_of_matcher.rb#L125

https://matchers.shoulda.io/docs/v5.3.0/Shoulda/Matchers/ActiveModel.html#validate_presence_of-instance_method

I couldn't find this written in the documentation but according to AI, validate_presence_of sets the value of the attribute it's testing to nil. Then for inclusion validations, it strictly checks for the default message unless told otherwise with with_message.